listings/ch00-introduction/constructor/src/constructor.cairo

```diff
@@ -0,0 +1,10 @@
+use starknet::ContractAddress;
```

Comment:
All of this must be wrapped inside a contract module.
When you run `scarb build` on the provided code, it fails, because you didn't wrap this code inside a contract.